src/libraries/System.Text.Json/src/System/Text/Json/Schema/JsonSchema.cs

```diff
@@ -260,6 +260,19 @@ JsonNode CompleteSchema(JsonNode schema)
             }
         }
 
+        public static void EnsureMutable(ref JsonSchema schema)
+        {
+            switch (schema._trueOrFalse)
+            {
+                case false:
+                    schema = new JsonSchema { Not = True };
+                    break;
+                case true:
+                    schema = new JsonSchema();
```

Comment:
Can you help me understand what this is doing?

Reply:
It's converting a boolean schema instance into an equivalent object-based schema so that additional keywords can be appended. Namely:

* `false` maps to `{ "not": true }` and
* `true` maps to `{ }`

Reply:
We want to default to `true` for universal schemas (e.g. for types like `object?` or `JsonElement`) but we need to fall back to an object-based representation if additional keywords are necessary, e.g. when a `default` value needs to specified in a parameter.

Reply:
thanks. Might be worth a comment.

Reply:
Done.